### ISSUE TITLE
Can't add release notes that start with a `-` char

### ISSUE THREAD

**Issue by ColtonIdle** [guness/bitrise-step-firebase-app-distribution#40]
The firebase command ends up resolving to

```
firebase appdistribution:distribute "/bitrise/deploy/app-release.apk" --app "1:123:android:456" --release-notes "- Add feature 1\n- Add feature 2\n" --groups "entirecompany"
```

which fails with `error: unknown option '-A'`

After reaching out to firebase support they recommended using 

```
--release-notes="- Add feature 1\n- Add feature 2\n"
```

which resolved the issue, but I have to script it by hand to do that. Would be nice if the firebase step was updated to support a leading `-` character in release notes.

**Comment by leoSTM** [comment]
Yeah, agree, it happened to me as well, and it really took me a while to figure it out as I never thought that is a problem

**Comment by aclima93** [comment]
This issue is still prevalent.
